agent/xds/testdata/endpoints/transparent-proxy-with-peered-upstreams.latest.golden

```diff
@@ -3,7 +3,7 @@
   "resources": [
     {
       "@type": "type.googleapis.com/envoy.config.endpoint.v3.ClusterLoadAssignment",
-      "clusterName": "api-a.default.default.cloud.external.1c053652-8512-4373-90cf-5a7f6263a994.consul",
+      "clusterName": "api-a.default.peer-a.external.1c053652-8512-4373-90cf-5a7f6263a994.consul",
```

Comment:
tproxy tests copy-pasted the PeerMeta from the connect proxy tests, so the clusters contained a reference to `cloud`. In the actual setup, the peer name is `peer-a` so this is more correct.